src/Analyzers/Core/Analyzers/NewLines/MultipleBlankLines/AbstractMultipleBlankLinesDiagnosticAnalyzer.cs

```diff
@@ -59,8 +59,8 @@ private void AnalyzeTree(SyntaxTreeAnalysisContext context, CompilationOptions c
             if (!context.ShouldAnalyzeSpan(child.FullSpan))
                 continue;
 
-            if (child.IsNode)
-                Recurse(context, notificationOption, child.AsNode()!, cancellationToken);
+            if (child.AsNode(out var childNode))
+                Recurse(context, notificationOption, childNode, cancellationToken);
             else if (child.IsToken)
```

Comment:
> else if (child.IsToken) [](http://example.com/codeflow?start=11&length=24)

else if (child.AsToken(out var childToken))

:)

Reply:
Maybe later :)  i wanted to stop the `AsNode()!` craziness.